solidity/random-beacon/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "@keep-network/random-beacon-v2",
+  "version": "0.0.1-dev",
```

Don't we want to start versioning with v2?
```suggestion
  "version": "2.0.0-dev",
```
cc @pdyraga

---

115717c